src/System.Net.Http/tests/FunctionalTests/HttpProtocolTests.cs

```diff
@@ -35,6 +32,17 @@ public class HttpProtocolTests : HttpClientTest
             await GetAsyncSuccessHelper(statusLine, expectedStatusCode, expectedReason);
         }
 
+        [Theory]
+        [InlineData("HTTP/1.1 200      ", 200, "     ")]
```

Comment:
Would it make sense to add into the test the behavior on other platforms? (more as documentation of differences) - i.e. "" for .NET Core on Windows. Not sure where .NET Framework / .NET Core on Linux stand.
For the case where we are violating RFC, it would make sense IMO.

Reply:
I will add comment to the test.

Reply:
What if you add one more arg -- the value which will be checked on those platforms?

Reply:
This is really a good idea. I will submit a new commit.